### ISSUE TITLE
Tumblr import fails `post_to_hash': undefined method `<<'…

### ISSUE THREAD

**Issue by jng5** [jekyll/jekyll-import#29]
I cannot import my blog. Doing:

```
$ ruby -rubygems -e 'require "jekyll/jekyll-import/tumblr";
>      JekyllImport::Tumblr.process("http://blog.julieng.me", "md", true)'
```
I get the following error message:
```
/Users/julieng/.rvm/gems/ruby-1.9.3-p194/gems/jekyll-import-0.1.0.beta1/lib/jekyll/jekyll-import/tumblr.rb:94:in `post_to_hash': undefined method `<<' for nil:NilClass (NoMethodError)
	from /Users/julieng/.rvm/gems/ruby-1.9.3-p194/gems/jekyll-import-0.1.0.beta1/lib/jekyll/jekyll-import/tumblr.rb:27:in `block in process'
	from /Users/julieng/.rvm/gems/ruby-1.9.3-p194/gems/jekyll-import-0.1.0.beta1/lib/jekyll/jekyll-import/tumblr.rb:27:in `map'
	from /Users/julieng/.rvm/gems/ruby-1.9.3-p194/gems/jekyll-import-0.1.0.beta1/lib/jekyll/jekyll-import/tumblr.rb:27:in `process'
	from -e:2:in `<main>'
```

It command works successfully for _other_ tumblrs, just not mine. Any ideas?

As far as I can tell, __I do not have any of the following which may be _problem_ post types__ (per google results and other issues):
* audio
* chats

I have the following post types:

* text
* photos
* videos
* links
* quotes
* reblogs
* "ask me anything" posts

Any ideas? Thanks for your help.

**Comment by paramaggarwal** [comment]
I faced the same problem, and @jng5's method above works well. Submitted a PR with the this fix: #179.

**Comment by octalmage** [comment]
PR #179 fixed this for me, thanks @paramaggarwal and @jng5!

**Comment by parkr** [comment]
Fixed by #179.

**Comment by paramaggarwal** [comment]
Thanks @parkr and @jng5.

**Comment by parkr** [comment]
This fix has been released in v0.7.0: https://github.com/jekyll/jekyll-import/releases/tag/v0.7.0
